C++/4. Advanced-Operators.md

```diff
@@ -76,7 +76,7 @@ Relational Operators are operators which compare things together:
 
 All of the above operators compare two values against each other. For example, `x == y` checks if `x` and `y` are equal.
 
-**Note: At the lowest level, computers represent booleans as an integer, '0' or '1' (for "false" and "true" respectively), and normally C++'s `std::cout <<` will quite literally show `0` or `1`. We can change this behavior by manipulating `std::cout`, by using `std::boolalpha`, which will now show `true` or `false`, instead of `1` or `0`. (`std::cout` is a `std::ostream`, streams will be covered much later)**
+**Note: At the lowest level, computers represent booleans as an integer, '0' or '1' (for "false" and "true" respectively), and normally C++'s `std::cout <<` will quite literally show `0` or `1`. We can change this behavior by manipulating `std::cout`, by using `std::boolalpha`, which will now show `true` or `false`, instead of `1` or `0`. (`std::cout` is a `std::ostream`; streams will be covered much later)**
```

Comment:
did you mean `std::cout` is a `std::iostream`?

Reply:
No, I meant ostream.

See the declaration of std::cout at https://en.cppreference.com/w/cpp/io/cout

Example:
https://replit.com/@xxpertHacker/JumpyWatchfulComments#main.cpp